en/docs/chapter_array_and_linkedlist/ram_and_cache.md

```diff
@@ -16,56 +16,56 @@ There are three types of storage devices in computers: <u>hard disk</u>, <u>rand
 
 Clearly, **the fewer the cache misses, the higher the CPU's data read-write efficiency**, and the better the program performance. The proportion of successful data retrieval from the cache by the CPU is called the <u>cache hit rate</u>, a metric often used to measure cache efficiency.
 
 To achieve higher efficiency, caches adopt the following data loading mechanisms.
 
-- **Cache lines**: Caches don't store and load data byte by byte but in units of cache lines. Compared to byte-by-byte transfer, the transmission of cache lines is more efficient.
-- **Prefetch mechanism**: Processors try to predict data access patterns (such as sequential access, fixed stride jumping access, etc.) and load data into the cache according to specific patterns to improve the hit rate.
-- **Spatial locality**: If data is accessed, data nearby is likely to be accessed in the near future. Therefore, when loading certain data, the cache also loads nearby data to improve the hit rate.
+- **Cache lines**: Caches operate by storing and loading data in units called cache lines, rather than individual bytes. This approach improves efficiency by transferring larger blocks of data at once.
+- **Prefetch mechanism**: Processors predict data access patterns (e.g., sequential or fixed-stride access) and preload data into the cache based on these patterns to increase the cache hit rate.
+- **Spatial locality**: When a specific piece of data is accessed, nearby data is likely to be accessed soon. To leverage this, caches load adjacent data along with the requested data, improving hit rates.
 - **Temporal locality**: If data is accessed, it's likely to be accessed again in the near future. Caches use this principle to retain recently accessed data to improve the hit rate.
 
-In fact, **arrays and linked lists have different cache utilization efficiencies**, mainly reflected in the following aspects.
+In fact, **arrays and linked lists have different cache utilization efficiencies**, which is mainly reflected in the following aspects.
 
-- **Occupied space**: Linked list elements occupy more space than array elements, resulting in less effective data volume in the cache.
-- **Cache lines**: Linked list data is scattered throughout memory, and since caches load "by line," the proportion of loading invalid data is higher.
-- **Prefetch mechanism**: The data access pattern of arrays is more "predictable" than that of linked lists, meaning the system is more likely to guess which data will be loaded next.
-- **Spatial locality**: Arrays are stored in concentrated memory spaces, so the data near the loaded data is more likely to be accessed next.
+- **Occupied space**: Linked list elements take up more space than array elements, resulting in less effective data being held in the cache.
+- **Cache lines**: The linked list data is scattered throughout the memory, and the cache is "loaded by row", so the proportion of invalid data loaded is higher.
```

Comment:
i don't think the "the"s added in this line is needed, because we are not referring to any specific item or scenario.